<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>730</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>1200</x>
       <y>0</y>
       <width>80</width>
       <height>730</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,80);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_screen_menu1">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>60</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe MDL2 Assets</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgba(0,0,0,80);
	color:rgba(255,255,255,210);
	border-radius:20px;
}
QPushButton:hover{
	color: rgba(131,96,53,255);
}
QPushButton:clicked{
	padding-left: 5px;
	padding-top: 5px;
	color:rgba(91,88,53,255);
}</string>
     </property>
     <property name="text">
      <string></string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_screen_menu2">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>160</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe MDL2 Assets</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgba(0,0,0,80);
	color:rgba(255,255,255,210);
	border-radius:20px;
}
QPushButton:hover{
	color: rgba(131,96,53,255);
}
QPushButton:clicked{
	padding-left: 5px;
	padding-top: 5px;
	color:rgba(91,88,53,255);
}</string>
     </property>
     <property name="text">
      <string></string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_screen_menu3">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>260</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe MDL2 Assets</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgba(0,0,0,80);
	color:rgba(255,255,255,210);
	border-radius:20px;
}
QPushButton:hover{
	color: rgba(131,96,53,255);
}
QPushButton:clicked{
	padding-left: 5px;
	padding-top: 5px;
	color:rgba(91,88,53,255);
}</string>
     </property>
     <property name="text">
      <string></string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_screen_menu4">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>360</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>배달의민족 주아</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgba(0,0,0,80);
	color:rgba(255,255,255,210);
	border-radius:20px;
}
QPushButton:hover{
	color: rgba(131,96,53,255);
}
QPushButton:clicked{
	padding-left: 5px;
	padding-top: 5px;
	color:rgba(91,88,53,255);
}</string>
     </property>
     <property name="text">
      <string>AP</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1200</width>
       <height>730</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label_slide_home">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>470</width>
         <height>264</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:gba(0,0,0,80);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_slide_home">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>470</width>
         <height>264</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgba(0,0,0,0);
}
QPushButton:hover{
}
QPushButton:clicked{
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget_home">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>350</y>
         <width>470</width>
         <height>300</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>11</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QTabWidget::pane {border: 0;}</string>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_home2">
        <attribute name="title">
         <string>주요소식</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_home2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>470</width>
           <height>270</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>date</string>
          </property>
         </column>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_home1">
        <attribute name="title">
         <string>공지사항</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget_home1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>470</width>
           <height>270</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <column>
          <property name="text">
           <string>title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>date</string>
          </property>
         </column>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>40</y>
         <width>391</width>
         <height>641</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,80);</string>
       </property>
       <property name="text">
        <string>트위터</string>
       </property>
      </widget>
      <zorder>label_slide_home</zorder>
      <zorder>tabWidget_home</zorder>
      <zorder>label_6</zorder>
      <zorder>pushButton_slide_home</zorder>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>1151</width>
         <height>701</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="styleSheet">
       <string notr="true">QListWidget::item:selected { background-color: white; }</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1200</width>
         <height>730</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(0,0,0,180)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_cal2">
       <property name="geometry">
        <rect>
         <x>425</x>
         <y>15</y>
         <width>250</width>
         <height>700</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerItem</enum>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_cal3">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>15</y>
         <width>250</width>
         <height>454</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerItem</enum>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_cal4">
       <property name="geometry">
        <rect>
         <x>935</x>
         <y>15</y>
         <width>250</width>
         <height>454</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerItem</enum>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_cal1">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>15</y>
         <width>380</width>
         <height>700</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerItem</enum>
       </property>
       <property name="batchSize">
        <number>100</number>
       </property>
      </widget>
      <widget class="QPushButton" name="button_calgrowth_insert">
       <property name="geometry">
        <rect>
         <x>395</x>
         <y>40</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe MDL2 Assets</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgba(255,255,255,80);
	color:rgba(0,0,255,240);
	border-radius:15px;
}
QPushButton:hover{
	color: rgba(131,96,53,255);
}
QPushButton:clicked{
	padding-left: 5px;
	padding-top: 5px;
	color:rgba(91,88,53,255);
}</string>
       </property>
       <property name="text">
        <string></string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_calgrowth_delete">
       <property name="geometry">
        <rect>
         <x>395</x>
         <y>80</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe MDL2 Assets</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgba(255,255,255,80);
	color:rgba(255,0,0,240);
	border-radius:15px;
}
QPushButton:hover{
	color: rgba(131,96,53,255);
}
QPushButton:clicked{
	padding-left: 5px;
	padding-top: 5px;
	color:rgba(91,88,53,255);
}</string>
       </property>
       <property name="text">
        <string></string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QPushButton" name="button_ap1">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>200</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe MDL2 Assets</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgba(0,0,0,210);
	color:rgba(255,255,255,210);
	border-radius:5px;
}
QPushButton:hover{
	color: rgba(131,96,53,255);
}
QPushButton:clicked{
	padding-left: 5px;
	padding-top: 5px;
	color:rgba(91,88,53,255);
}
</string>
       </property>
       <property name="text">
        <string></string>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit_ap1">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>137</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="time">
        <time>
         <hour>11</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_ap1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>137</y>
         <width>160</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="time">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
       <property name="displayFormat">
        <string>yyyy/MM/dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_ap2">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>140</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>점검 시간</string>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit_ap2">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>137</y>
         <width>135</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="time">
        <time>
         <hour>16</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
      <widget class="QLabel" name="label_ap3">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>140</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>~</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ap1">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>90</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>이벤트명</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_ap1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>87</y>
         <width>300</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_ap4">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>190</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>여유 시간</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_ap1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>187</y>
         <width>51</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_ap5">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>60</y>
         <width>387</width>
         <height>567</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(0,0,0,50)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="button_ap2">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>200</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgba(0,0,0,210);
	color:rgba(255,255,255,210);
	border-radius:5px;
}
QPushButton:hover{
	color: rgba(131,96,53,255);
}
QPushButton:clicked{
	padding-left: 5px;
	padding-top: 5px;
	color:rgba(91,88,53,255);
}
</string>
       </property>
       <property name="text">
        <string>경로</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ap6">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>315</y>
         <width>200</width>
         <height>100</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>배달의민족 주아</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,0);
color:rgba(255,255,255,0);
border-radius:20px;</string>
       </property>
       <property name="text">
        <string>저장 성공</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1280</width>
       <height>730</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/newPrefix/Useimages/28.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>stackedWidget</zorder>
    <zorder>label_2</zorder>
    <zorder>button_screen_menu1</zorder>
    <zorder>button_screen_menu2</zorder>
    <zorder>button_screen_menu3</zorder>
    <zorder>button_screen_menu4</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
